demos/wasmcloud/uppercase/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+use log::info;
+use serde::Serialize;
+use wapc_guest::prelude::*;
+use wasmcloud_actor_http_server as http;
+
+#[no_mangle]
+pub fn wapc_init() {
+    wasmcloud_actor_core::Handlers::register_health_request(health);
+    http::Handlers::register_handle_request(uppercase);
+}
```

Comment:
same comment as above for the `init` function (again, replacing it with the init won't change the functionality you have, it'll just remove some of the boilerplate you have to put in for the health check)